SPECS/hootenanny.spec

```diff
@@ -223,16 +220,6 @@ popd
 %{__install} -d -m 0775 %{buildroot}%{tomcat_webapps}/%{name}-id
 %{__install} -d -m 0775 %{buildroot}%{tomcat_webapps}/%{name}-id/data
 %{__cp} -pr hoot-ui-2x/dist/* %{buildroot}%{tomcat_webapps}/%{name}-id/
-%{__install} -m 0644 hoot-ui/data/osm-plus-taginfo.csv %{buildroot}%{tomcat_webapps}/%{name}-id/data
-%{__install} -m 0644 hoot-ui/data/tdsv61_field_values.json %{buildroot}%{tomcat_webapps}/%{name}-id/data
```

Comment:
Are these two files no longer needed/used? I see they were coming from `hoot-ui`, but they were being installed into `hoot-ui-2x`'s data directory (hence being outside of the `%if %{with hoot_ui_legacy}` block) so might have been used by it in some cases. Am I understanding this correctly or was their initial inclusion unnecessary or something like that?

Reply:
Good catch!
hoot-ui-2x did use them up until this recent PR https://github.com/ngageoint/hootenanny-ui/pull/2032
so we must have overloaded this PR to account for that while removing the legacy ui